hfile.c

```diff
@@ -585,7 +598,11 @@ hFILE *hdopen(int fd, const char *mode)
 static hFILE *hopen_fd_fileuri(const char *url, const char *mode)
 {
     if (strncmp(url, "file://localhost/", 17) == 0) url += 16;
+#ifdef _WIN32
+    else if (strncmp(url, "file:///", 8) == 0) url += 8;
+#else
```

Comment:
Surely a little obscure, and doesn't handle `file://localhost/C:/foo/bar.txt`.  Clearer alternative would be adding something like this below:

    #ifdef _WIN32
    if (url[0] == '/' && url[2] == ':' && url[3] == '/') url += 1;
    #endif